### ISSUE TITLE
Crash if hosts file doesn't exist on Windows 10

### ISSUE THREAD

**Issue by Spartelfant** [cogitatio/vagrant-hostsupdater#178]
My hosts file had gotten messy (unrelated to hostsupdater) so I figured the simplest remedy was to delete it, assuming hostsupdater would recreate it if needed.

But instead `%WINDIR%\System32\drivers\etc\hosts` not existing lead to hostsupdater crashing, in turn causing vagrant to fail during provisioning. I've attached a textfile with the error output. As a sidenote, I was running vagrant from an elevated command prompt.

[hostsupdater error output.txt](https://github.com/cogitatio/vagrant-hostsupdater/files/3137529/hostsupdater.error.output.txt)

Please let me know if I can contribute in any way to solving this or narrowing down the cause.

**Comment by cgsmith** [comment]
@Spartelfant good catch on this bug. I will make a check for a hosts file before attempting a write.
